package.json

```diff
@@ -12,7 +12,7 @@
     "vetur"
   ],
   "scripts": {
-    "build": "siroc build && mkdist --src src/runtime --dist dist/runtime -d --ext js",
+    "build": "siroc build && mkdist --src src/runtime --dist dist/runtime -d --ext js && rollup -c",
```

Comment:
Shall we use unbuild? :))

Reply:
Its time has come! 😁

Reply:
(I'm thinking using unbuild should be a separate PR though.)